lib/ansible/modules/source_control/github_webhook.py

```diff
@@ -0,0 +1,240 @@
+#!/usr/bin/python
+#
+# Copyright: (c) Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: github_webhook
+short_description: Manage Github webhooks
+version_added: "2.6"
+description:
+  - "Create and delete Github webhooks"
+requirements:
+  - "PyGithub >= 1.3.5"
+options:
+  repository:
+    description:
+      - Full name of the repository to configure a hook for
+    required: true
+  url:
+    description:
+      - URL to which payloads will be delivered
+    required: true
+  content_type:
+    description:
+      - The media type used to serialize the payloads
+    required: false
+    choices: [ form, json ]
+    default: form
+  secret:
+    description:
+      - The shared secret between Github and the payload URL.
+    required: false
+  insecure_ssl:
```

Comment:
Can you rename this to `validate_certs` to match Ansible parameter naming convention ?

Reply:
I'd really rather not, for two reasons:

1. That introduces an asymmetricity between the inputs and outputs of the webhook modules. See the previous discussion on this PR for details.

2. This doesn't do the same thing as other `validate_certs` options. I've clarified the documentation.